packages/xod-arduino/src/transpiler.js

```diff
@@ -94,6 +94,11 @@ const arrangeTPatchesInTopologicalOrder = def(
     )(project)
 );
 
+const commentXodPragmas = def(
+  'commentXodPragmas :: String -> String',
+  R.replace(/#\s*pragma\s+XOD\s/, '//#pragma XOD ')
```

Comment:
:smiling_imp: 

Will it also comment the following?

```cpp
Serial.println("#pragma XOD hello")
```

It must not. Suggest prefixing with the beginning of the line + possible whitespace before `#`.

By the way, I see the small issue is big enough to put a few unit tests on it.